README.md

```diff
@@ -107,6 +107,22 @@ In the example content below, the first row would cause any payouts to the first
 
 To enable these features, create a file src/data/.substitutePayTo and configure according to your situation.  
 
+## Using payor specific Fees (New!)
+You can use payor specific fees. They will override the value of `SEND_TRANSACTION_FEE` based on the public key specified. To use this feature, 
```

Comment:
This should be overriding the pool commission, not send transaction fee

Reply:
I apologize for the confusion - I realize it should have been talked about as pool commission, not fee. The variable that we want to override, on a per-key basis, is commissionRate in the calculator.